CONTRIBUTING.md

```diff
@@ -30,6 +30,13 @@ Dependencies that are added to the `package.json` file at the root of the
 repo are shared between packages. Dependencies listed in the
 `package.json` files within each individual package directory work as normal.
 
+### For Windows users
+There is a known [bug](https://github.com/lerna/lerna/issues/385#issuecomment-948402686) which running `lerna bootsrap` fails to generate the `pwa-kit-cli` bin shims
```

Comment:
Gonna need to wait for the new name of `pwa-kit-cli` come into place before merging

Reply:
It looks like `pwa-kit-dev` is the new name.